test/e2e/multihoming.go

```diff
@@ -863,7 +864,7 @@ var _ = Describe("Multi Homing", func() {
 								[]mnpapi.MultiPolicyType{mnpapi.PolicyTypeIngress, mnpapi.PolicyTypeEgress},
 								nil,
 								[]mnpapi.MultiNetworkPolicyEgressRule{
-									{},
+									mnpapi.MultiNetworkPolicyEgressRule{},
```

Comment:
why do we need this ? 

Maybe isolate this in a separate commit providing the motivation in the commit msg ?